Add documentation about request_body directive in Caddyfile (#104)
caddyserver/website: src/docs/markdown/caddyfile/directives/request_body.md

---
title: request_body (Caddyfile directive)
---

# request_body

> Manipulates request body.

Reviewer: ```suggestion
Manipulates or sets restrictions on the bodies of incoming requests.
```

## Syntax

```caddy-d

> request_body {

Reviewer 2: This should note that it supports matchers

```suggestion
request_body [<matcher>] {
```

  max_size <value>
}
```

> - **&lt;value&gt;** is the maximum size in bytes allowed for the request body. It accepts all formats supported by [go-humanize](https://github.com/dustin/go-humanize/blob/master/bytes.go).

Reviewer: ```suggestion
- **max_size** is the maximum size in bytes allowed for the request body. It accepts all formats supported by [go-humanize](https://github.com/dustin/go-humanize/blob/master/bytes.go).
```

## Examples

> Limit request body s to 10 mega bytes:

Reviewer: ```suggestion
Limit request body sizes to 10 megabytes:
```
